lib/screens/play/current_card/card_actions/card_action_cost.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+class CardActionCost extends StatelessWidget {
+  final List<Resource> cost;
+
+  CardActionCost([this.cost = const []]);
```

Comment:
I think we can drop the optional / default as we're always passing something in?